src/libs/antares/study/area/list.cpp

```diff
@@ -1535,6 +1535,9 @@ bool AreaList::renameArea(const AreaName& oldid, const AreaName& newid, const Ar
         assert(oldCount == a.links.size() and "We must have the same number of items in the list");
 #endif
     });
+    
+    // gp : do we really need to rebuild the area's link indexes here ?
+    area->buildLinksIndexes();
```

Comment:
Since `a` in the loop above could take the value `area`, its links could be changed at line 1532. So I think it is necessary to rebuild index here.